docs/assets/0123-import-scripts/README.md

```diff
@@ -0,0 +1,142 @@
+# Welcome to 0123-import-scripts!
```

Comment:
Is there a reason for having this "sample repo" as part of the dsp-tools repo, rather than a separate github repo? if it were a separate github repo, there would be the following advantages that I can spontaneously think of:
- downloading as ZIO would come out of the box
- the README.md would then be the top level readme, so it would get displayed on github right away
- when cloning the repo, users would have it as a separate repo on their file system
- if recommending GitHub Desktop, cloning the repo would be extremely easy for users; and there would even be a "open in Visual Studio Code" button